mysite/_config/database.yml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
This should have been in the 4.7.0 upgrade, but it looks like I forgot to do a `git add`.